okhttp/src/main/java/okhttp3/internal/connection/RealConnectionPool.kt

```diff
@@ -111,6 +111,7 @@ class RealConnectionPool(
       return true
     } else {
       // Awake the cleanup thread: we may have exceeded the idle connection limit.
+      @Suppress("PLATFORM_CLASS_MAPPED_TO_KOTLIN")
```

Comment:
Maybe create extensions for these? And suppress all in one place rather than littering callsites.